### PR TITLE
Feat(#8ug661): add notifications tab

### PR DESCRIPTION
**Pull request by dannylekim**

### CODE REVIEW

**Comment by dannylekim** [comment]
Task linked: [CU-8ug661 View pending Bills](https://app.clickup.com/t/8ug661)

**Comment by dannylekim** [comment]
the duplicated code smell makes no sense, I think we can ignore it

File: src/components/PendingBillCard/PendingBillCard.test.jsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        );
+      });
+    });
+
+    describe("component ", () => {
+      it("should not find an active bill", () => {
+        wrapper = shallow(
+            <PendingBillCard
+                bill={bill}
+                activeBillId={999}
```
Comment (alphakennyn) [inline src/components/PendingBillCard/PendingBillCard.test.jsx]: Why don't you do this test as snapshot? IN the snapshot, we'll be expecting `.non__active__bill` to appear as well

File: src/components/PendingBillCard/PendingBillCard.test.jsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+                acceptBillInvitationHandler={jest.fn()}
+                declineBillInvitationHandler={jest.fn()}
+                filterDateHandler={jest.fn()}
+                getBillIconHandler={jest.fn()}
+            />
+        );
+        expect(wrapper.find(".non__active__bill")).toHaveLength(1);
+      });
+
+      it("should find an active bill", () => {
```
Comment (alphakennyn) [inline src/components/PendingBillCard/PendingBillCard.test.jsx]: I find this test a little redundant since the test case is covered by the snapshot

**Review by alphakennyn — APPROVED** [review]

**Pull request merged**